package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "5.11.3",
+  "version": "5.11.4-rc.4",
```

Comment:
When does this become 5.11.4?

Reply:
When we make the actual release, the version will be updated to `"5.11.4"` here.